packages/design-system/src/components/Autocomplete/Autocomplete.test.tsx

```diff
@@ -40,6 +40,70 @@ describe('Autocomplete', () => {
     expect(items.text()).toEqual('Cook County, IL');
   });
 
+  it('renders items with children property', () => {
+    const items = [
+      {
+        id: '1',
+        name: 'Carrots (1)',
+        children: (
+          <>
+            Carrots <strong>(1)</strong>
+          </>
+        ),
+      },
+      {
+        id: '2',
+        name: 'Cookies (3)',
+        children: (
+          <>
+            Cookies <strong>(3)</strong>
+          </>
+        ),
+      },
+      {
+        id: '3',
+        name: 'Crackers (2)',
+        children: (
+          <>
+            Crackers <strong>(2)</strong>
+          </>
+        ),
+      },
+      {
+        id: '4',
+        children: <a href="https://duckduckgo.com/?q=snacks">Search all snacks</a>,
+      },
+    ];
+
+    const tree = renderer
+      .create(
+        <Autocomplete items={items} isOpen>
+          <TextField label="autocomplete" name="autocomplete_field" />
+        </Autocomplete>
+      )
+      .toJSON();
+
+    // TODO: Clean this up by using another rendering library or figure out how to use this one to
```

Comment:
[suggestion] Consider using enzyme for this test to be more consistent with other unit tests. Starting at line 78, it would look like
```
    const wrapper = mount(
        <Autocomplete items={items} isOpen>
          <TextField label="autocomplete" name="autocomplete_field" />
        </Autocomplete>
    );
    const ul = wrapper.find('ul');
    expect(ul).toMatchSnapshot();
```
I tested ☝🏻 locally and I think it works, but if it gives you any issues, your current tests works.

Reply:
Also, separate issue, not for this PR, but I don't understand why we use both `react-test-renderer` & `enzyme`. What does one provide that the other doesn't?

Reply:
> I don't understand why we use both react-test-renderer & enzyme

In this case I think we have both because when enzyme wrappers are serialized, they aren't human-readable. In other products, we use `enzyme-to-json` to get an HTML-looking snapshot. We could add it. What do you think? You can opt-in to using it in specific modules rather than adopting it for all.

Reply:
I take that back! It looks like the serialized Enzyme objects are pretty now. That must have changed since I last tried using them in snapshots natively.